apps/emqx_bridge/src/emqx_bridge_v2_api.erl

```diff
@@ -70,7 +70,11 @@
 namespace() -> "bridge_v2".
 
 api_spec() ->
-    emqx_dashboard_swagger:spec(?MODULE, #{check_schema => true}).
+    %% TODO
+    %% The check_schema option needs to be set to false so we get the
+    %% query_string to the delete operation. We can change this once
+    %% we have fixed the schmea for the delete operation.
+    emqx_dashboard_swagger:spec(?MODULE, #{check_schema => false}).
```

Comment:
I think we shouldn't disable the schema check here.  Having unexpected terms passed down to the handlers is confusing, and not being able to rely on the type checking/coercion makes things harder.

I think we just have to add the expected parameters to the `parameters` key in the API spec schema.